test/groovy/MTABuildTest.groovy

```diff
@@ -30,95 +30,52 @@ public class MTABuildTest extends BasePipelineTest {
 
 
     def currentDir
-    def otherDir
-    def mtaBuildShEnv
-
     def mtaBuildScript
-    def cpe
+
 
     @Before
     void init() {
 
         currentDir = tmp.newFolder().toURI().getPath()[0..-2] //omit final '/'
-        otherDir = tmp.newFolder().toURI().getPath()[0..-2] //omit final '/'
 
-        helper.registerAllowedMethod('readYaml', [Map], {
-            m ->
-                return new Yaml().load((m.file as File).text)
-        })
-        helper.registerAllowedMethod("dir", [String, Closure], {
-            s, c ->
-                currentDir = "${currentDir}/${s}"
-                c()
-        })
         helper.registerAllowedMethod('pwd', [], { currentDir } )
-        helper.registerAllowedMethod("withEnv", [List.class, Closure.class],
-                { l, c ->
-                    mtaBuildShEnv = l
-                    c()
-                })
         helper.registerAllowedMethod('error', [String], { s -> throw new hudson.AbortException(s) })
+        helper.registerAllowedMethod('fileExists', [String], { String path ->
+            return path.endsWith('.pipeline/config.yml')
+        })
 
         binding.setVariable('PATH', '/usr/bin')
-        binding.setVariable('JAVA_HOME', '/opt/java')
-        binding.setVariable('env', [:])
 
-        mtaBuildScript = loadScript("mtaBuild.groovy").mtaBuild
-        cpe = loadScript('commonPipelineEnvironment.groovy').commonPipelineEnvironment
+        mtaBuildScript = loadScript('mtaBuild.groovy').mtaBuild
     }
 
 
     @Test
-    public void straightForwardTest(){
-
-        binding.getVariable('env')['MTA_JAR_LOCATION'] = '/opt/mta'
+    public void environmentPathTest(){
```

Comment:
In principle nothing against renaming the tests, but this is out of scope here. The scope of this pull request is: "mtaBuild with new config framework". This makes it harder to review. Introducing better names for the test should be done in a dedicated PR.